chasm/src/main/java/org/quiltmc/chasm/api/Lock.java

```diff
@@ -1,5 +1,8 @@
 package org.quiltmc.chasm.api;
 
+/**
+ * Todo.
```

Comment:
I'm guessing this is the last missing public javadoc then? Let's make sure to not forget about it :)